src/style.css

```diff
@@ -0,0 +1,124 @@
+
+* {
+  padding: 0;
+  margin: 0;
+  background-color: #1c1a19;
+  text-decoration: none;
+  list-style: none;
+  box-sizing: border-box;
+}
+
+/* stylings for navigation */
+nav {
+  position: absolute;
+  height: 10%;
+  width: 100%;
+  background-color: #3c3a39;
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo-container {
+  display: flex;
+  align-items: center;
+  padding-top: 2.6%;
+  padding-left: 3%;
+  color: #fff;
+  background-color: #3c3a39;
+}
+
+#logo,
+#ham {
+  background-color: #3c3a39;
+}
+
+.hamberger-btn {
+  background-color: #3c3a39;
+  display: block;
+  font-size: 25px;
+  color: white;
+  float: right;
+  line-height: 80px;
+  margin-right: 40px;
+  cursor: pointer;
+}
+
+.section {
+  background-color: #1c1a19;
+  height: 100%;
+}
+
+.container {
+  padding-top: 15%;
+  flex-grow: 1;
+  height: 100vh;
+}
+
+#my_name {
+  font-family: 'Times New Roman', Times, serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 540%;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+  flex: none;
+  padding-top: 18%;
+  padding-left: 5%;
+  align-items: center;
+}
+
+#title {
+  color: #fff5e1;
+  padding-left: 5%;
+  padding-top: 3%;
+  font-family: 'Times New Roman', Times, serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+}
+
+#describe_me {
+  color: #fff;
+  font-family: 'Times New Roman', Times, serif;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 110%;
+  padding-left: 5%;
+  padding-top: 3%;
+}
+
+.socialIcons {
+  display: flex;
+  padding-left: 8%;
+  align-items: center;
+  gap: 1.5em;
+  cursor: pointer;
+  list-style-type: none;
+  margin-top: 32px;
+}
```

Comment:
- Kindly for the social icons use appropriate color, you can import the white icons images instead of the grey ones or import imports from the library for icons, so you can be able to give them the color you want:+1:  it will make your design stick more to the figma design as required in the project :+1: 

## Figma design 

![image](https://user-images.githubusercontent.com/95297251/235138717-fcafe675-5d79-4d70-a814-7c39640ee304.png)

## Your design

![image](https://user-images.githubusercontent.com/95297251/235138584-03a10f95-c447-41ef-a718-c386d386fc27.png)